BUILDING.md

```diff
@@ -15,22 +15,13 @@ Optionally build a specific version
 ./build-image.sh 0.3
 ```
 
-## Publishing
+## Deployment
 
-The `push-images.sh` script will push the docker images for all the versioned folders in the
-repository.
+We use continuous deployment via Travis to push images to Docker Hub.
```

Comment:
nit `Travis CI`

Reply:
Oh right, I always call it `Travis`, but `Travis CI` is actually the correct name.